Tests/UnitTests/Purchasing/Purchases/BasePurchasesTests.swift

```diff
@@ -210,22 +212,19 @@ extension BasePurchasesTests {
         var userID: String?
         var originalApplicationVersion: String?
         var originalPurchaseDate: Date?
-        var timeout = false
         var getSubscriberCallCount = 0
         var overrideCustomerInfoResult: Result<CustomerInfo, BackendError> = .success(
             // swiftlint:disable:next force_try
             try! CustomerInfo(data: BasePurchasesTests.emptyCustomerInfoData)
         )
 
         override func getCustomerInfo(appUserID: String, completion: @escaping Backend.CustomerInfoResponseHandler) {
-            getSubscriberCallCount += 1
-            userID = appUserID
+            self.getSubscriberCallCount += 1
+            self.userID = appUserID
 
-            if !timeout {
```

Comment:
This property made it so that this method wouldn't invoke the callback, which made the tests not actually do anything.
Instead, I've updated the tests to ensure that when the callback is called, the data is correct.